### PR TITLE
[OV JS] Support add_extension method

### PR DESCRIPTION
**Pull request by vishniakov-nikolai**
### Tickets:
 - 137673

### CODE REVIEW

File: src/bindings/js/node/tests/core.test.js [hunk 1]
```diff
@@ -43,11 +43,25 @@ it('Core.setProperty(\'CPU\', { \'NUM_STREAMS\': 5 })', () => {
   assert.equal(result, streams);
 });
 
-it('Core.setProperty(\'CPU\', { \'INFERENCE_NUM_THREADS\': 3 })', () => {
```
Comment (almilosz) [inline src/bindings/js/node/tests/core.test.js:46]: Let's fix old code-style issues in a separate PR dedicated to eslint fixes

**Review by almilosz — COMMENTED** [review]
I don't see a test case for a successful scenario.  Is this task a part of some other bigger task/activity?

File: src/bindings/js/node/tests/core.test.js (hunk 1 above)
Comment (vishniakov-nikolai) [reply src/bindings/js/node/tests/core.test.js:46]: My IDE did this automatically (removed trail spaces), do you want to rollback these changes?

**Comment by vishniakov-nikolai** [comment]
> I don't see a test case for a successful scenario. Is this task a part of some other bigger task/activity?

Yes, this is attempt to support latest OpenVINO feature https://github.com/openvinotoolkit/openvino_tokenizers
Usage example: https://github.com/openvinotoolkit/openvino.genai/blob/master/text_generation/causal_lm/cpp/beam_search_causal_lm.cpp#L156

File: src/bindings/js/node/src/core_wrap.cpp [hunk 2]
```diff
@@ -333,3 +334,15 @@ Napi::Value CoreWrap::get_property(const Napi::CallbackInfo& info) {
 
     return any_to_js(info, value);
 }
+
+void CoreWrap::add_extension(const Napi::CallbackInfo& info) {
+    try {
+        if (!info[0].IsString())
+            OPENVINO_THROW("addExtension method applies one argument of string type");
```
[the remark this answers is not in the document]
Comment (vishniakov-nikolai) [reply src/bindings/js/node/src/core_wrap.cpp:341]: This message will appear from JS side. From JS side method name is `addExtension`

**Pull request merged**